src/components/DecisioningEngine/createContextProvider.js

```diff
@@ -74,7 +77,8 @@ export default ({ eventRegistry, window }) => {
     return {
       ...coreGlobalContext,
       ...getTimeContext(),
-      window: getWindowContext()
+      window: getWindowContext(),
+      "~sdkver": version
```

Comment:
The version can be imported from `src/constants/libraryVersion.js`. No need to add a dependency and plugin to the build.

Reply:
Thanks @carterworks  for bringing this up. I was also looking for existing use, guess didn't search with matching keyword.

Reply:
+1